app/templates/_stylesheets.html

```diff
@@ -1,3 +1,5 @@
+<link rel="stylesheet" type="text/css" href="/bundle/pancake.css"/>
```

Comment:
where do these files come from?

Reply:
generated by pancake in the react frontend